chirp/drivers/ga510.py

```diff
@@ -285,6 +278,30 @@ def get_features(self):
                           (400000000, 480000000)]
         return rf
 
+    def validate_memory(self, mem):
+        msgs = chirp_common.CloneModeRadio.validate_memory(self, mem)
```

Comment:
This is really the python2 syntax for calling the parent, and I'm so used to the old way I forget that it's technically wrong now. But it really should be `super().validate_memory(mem)` in python3. It works, but the python2 way could stop working at some point in the python3.x timeline, so the fewer things we add, the better.

Reply:
OK. I will make another list. ;-)

Reply:
I know, it's annoying. Being through 15 years of python development it's hard to keep track of seemingly senseless changes like this. They all have reasons, but sometimes it's hard to be on board with it all :)

Reply:
Not annoyed. I'm happy to find out now. List made. There are 8 drivers that need attention. Nothing urgent. Should be easy to do. My kind of task. ;)